source/Octopus.Tentacle/Services/Scripts/Kubernetes/KubernetesScriptServiceV1Alpha.cs

```diff
@@ -169,14 +170,54 @@ static KubernetesScriptStatusResponseV1Alpha GetResponseForMissingScriptPod(Scri
                     new(ProcessOutputSource.StdErr, $"If you are using the default NFS storage, then also check if:"),
                     new(ProcessOutputSource.StdErr, $"- The NFS Pod was evicted due to exceeding its storage quota"),
                     new(ProcessOutputSource.StdErr, $"- The NFS Pod was restarted or moved as part of routine operation"),
-
-                }, 
+                },
                 lastLogSequence);
         }
 
         public bool IsRunningScript(ScriptTicket ticket)
         {
             return podStatusProvider.TryGetTrackedScriptPod(ticket) is not null;
         }
+
+        public async Task<KubernetesScriptStatusResponseV1Alpha> StartScriptAsync(StartKubernetesScriptCommandV1Alpha command, CancellationToken cancellationToken)
+            => (await StartScriptAsync(command.ToV1(), cancellationToken)).ToV1Alpha();
+
+        public async Task<KubernetesScriptStatusResponseV1Alpha> GetStatusAsync(KubernetesScriptStatusRequestV1Alpha request, CancellationToken cancellationToken)
+            => (await GetStatusAsync(new KubernetesScriptStatusRequestV1(request.ScriptTicket, request.LastLogSequence), cancellationToken)).ToV1Alpha();
+
+        public async Task<KubernetesScriptStatusResponseV1Alpha> CancelScriptAsync(CancelKubernetesScriptCommandV1Alpha command, CancellationToken cancellationToken)
+            => (await CancelScriptAsync(new CancelKubernetesScriptCommandV1(command.ScriptTicket, command.LastLogSequence), cancellationToken)).ToV1Alpha();
+
+        public Task CompleteScriptAsync(CompleteKubernetesScriptCommandV1Alpha command, CancellationToken cancellationToken)
+            => CompleteScriptAsync(new CompleteKubernetesScriptCommandV1(command.ScriptTicket), cancellationToken);
+    }
+
+    public static class CommandExtensions
+    {
+        public static StartKubernetesScriptCommandV1 ToV1(this StartKubernetesScriptCommandV1Alpha command)
+        {
+            return new StartKubernetesScriptCommandV1(
+                command.ScriptTicket,
+                command.TaskId,
+                command.ScriptBody,
+                command.Arguments,
+                command.Isolation,
+                command.ScriptIsolationMutexTimeout,
+                command.IsolationMutexName ?? "RunningScript", //In practice, this is never null due to the ExecuteScriptCommand.IsolationConfiguration.MutexName is not nullable
```

Comment:
Why do we need to coalesce here? Aren't the commands identical?

Reply:
So for some reason, the `IsolationMutextName` property is nullable, however the ctor parameter that populates this is not (it must have been an oversight).

I changed the V1 contract to have a non-nullable property, so it all matches up. This coalesce was a way to fix this without changing the `V1Alpha` contract 🤔